### PR TITLE
Added additional info about deleting stored payment method

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional info about deleting stored payment method from customer account dashboard

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-stored-payment-methods.html

### CODE REVIEW

File: src/customers/account-dashboard-stored-payment-methods.md [hunk 1]
```diff
@@ -27,4 +27,16 @@ _Stored Payment Methods_
 
 1. Clicks <span class="btn">Place Order</span>.
 
-The saved payment method is then displayed in the _Stored Payment Methods_ tab of the customer dashboard.
\ No newline at end of file
+The saved payment method is then displayed in the _Stored Payment Methods_ tab of the customer dashboard.
+
+## Delete a stored payment method
+
+Any of added earlier stored payment methods can not be edited by customer, but can be deleted only. This action cannot be undone.
```
Comment (BarnyShergold) [inline src/customers/account-dashboard-stored-payment-methods.md:34]:
```suggestion
Any previously added, stored payment methods cannot be edited by the customer, they can only be deleted. This action cannot be undone.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few grammatical corrections.

File: src/customers/account-dashboard-stored-payment-methods.md (hunk 1 above)
Comment (DmitryTsymbal) [reply src/customers/account-dashboard-stored-payment-methods.md]: Thank you for the review. Changes applied

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**